homework1/task03.py

```diff
@@ -0,0 +1,26 @@
+"""
+Write down the function, which reads input line-by-line, and find maximum and minimum values.
+Function should return a tuple with the max and min values.
+
+For example for [1, 2, 3, 4, 5], function should return [1, 5]
+
+We guarantee, that file exists and contains line-delimited integers.
+
+To read file line-by-line you can use this snippet:
+
+with open("some_file.txt") as fi:
+    for line in fi:
+        ...
+
+"""
+from typing import Tuple
+
+
+def find_maximum_and_minimum(file_name: str) -> Tuple[int, int]:
+    result = ()
+
+    with open(file_name, "r") as file:
+        for line in file:
+            file = list(line.replace(" ", "").split(","))
+            result = (int(min(file))), (int(max(file)))
+            return result
```

Comment:
You don't need to create a tuple separeatly, just `return min, max`

Reply:
I don't need it.